Tools/YamlCreate.ps1

```diff
@@ -25,1032 +29,1714 @@ $ScriptHeader = '# Created using YamlCreate.ps1 v1.1.6'
+                [System.IO.File]::WriteAllLines($($AppFolder + '\' + $DifLocale.Name), $MyRawString, $Utf8NoBomEncoding)
+            }
         }
     }
+
+    Write-Host 
+    Write-Host "Yaml file created: $LocaleManifestPath"
 }
 
-Function Submit-Manifest {
-    if (Get-Command 'git.exe' -ErrorAction SilentlyContinue) {
-        Write-Host
-        Write-Host
-        Write-Host -ForegroundColor 'White' "Submit PR?"
-        Write-Host "Do you want to submit your PR now?"
-        Write-Host -ForegroundColor 'Yellow' -NoNewline '[Y] Yes  '
-        Write-Host -ForegroundColor 'White' -NoNewline "[N] No "
-        Write-Host -NoNewline "(default is 'Y'): "
-        do {
-            $keyInfo = [Console]::ReadKey($false)
-        } until ($keyInfo.Key)
 
-        switch ($keyInfo.Key) {
-            'Y' {$PromptSubmit = '0'}
-            'N' {$PromptSubmit = '1'}
-            default {$PromptSubmit = '0'}
+Function Read-PreviousWinGet-Manifest-Yaml {
+    
+    if (($Option -eq 'NewLocale') -or ($Option -eq 'EditMetadata')) {
+        if (Test-Path -Path "$AppFolder\..\$PackageVersion") {
+            $script:OldManifests = Get-ChildItem -Path "$AppFolder\..\$PackageVersion"
+            $LastVersion = $PackageVersion
+        }
+        while (-not ($OldManifests.Name -like "$PackageIdentifier*.yaml")) {
+            Write-Host
+            Write-Host -ForegroundColor 'Red' -Object 'Could not find required manifests, input a version containing required manifests or "exit" to cancel'
+            $PromptVersion = Read-Host -Prompt 'Version' | TrimString
+            if ($PromptVersion -eq 'exit') { exit 1 }
+            if (Test-Path -Path "$AppFolder\..\$PromptVersion") {
+                $script:OldManifests = Get-ChildItem -Path "$AppFolder\..\$PromptVersion" 
+            }
+            $LastVersion = $PromptVersion
+            $script:AppFolder = (Split-Path $AppFolder) + "\$LastVersion"
+            $script:PackageVersion = $LastVersion
         }
     }
 
-    if ($PromptSubmit -eq '0') {
-        switch ($Option) {
-            'New' {$CommitType = 'New'}
-            'Update' {$CommitType = 'Update'}
-            'NewLocale' {$CommitType = 'Locale'}
+    if (-not (Test-Path -Path "$AppFolder\..")) {
+        $script:OldManifestType = 'None'
+        return
+    }
+    
+    if (!$LastVersion) {
+        try {
+            $script:LastVersion = Split-Path (Split-Path (Get-ChildItem -Path "$AppFolder\..\" -Recurse -Depth 1 -File -Filter '*.yaml').FullName ) -Leaf | Sort-Object $ToNatural | Select-Object -Last 1
+            $script:ExistingVersions = Split-Path (Split-Path (Get-ChildItem -Path "$AppFolder\..\" -Recurse -Depth 1 -File -Filter '*.yaml').FullName ) -Leaf | Sort-Object $ToNatural | Select-Object -Unique
+            Write-Host -ForegroundColor 'DarkYellow' -Object "Found Existing Version: $LastVersion"
+            $script:OldManifests = Get-ChildItem -Path "$AppFolder\..\$LastVersion"
```

Comment:
```suggestion
            $script:OldManifests = Get-ChildItem -Path "$AppFolder\..\$LastVersion" -Directory
```
To resolve #27518.

Reply:
I think this PR will take time to merge so we should open another PR for the time being so the users using YamlCreate.ps1 v1.1.6 wouldn't run into any issues.

Reply:
#27522